documentation/sphinx/source/disk-snapshot-backup.rst

```diff
@@ -0,0 +1,91 @@
+Field Name                        Description                                                Source of information
+================================  ========================================================   ========================================================
+``UID``                           unique identifier passed with all the                      ``snapshot`` CLI command output contains the UID
+                                  snapshot create binary invocations associated with
+                                  a backup. Disk snapshots could be tagged with this UID.
+``FoundationDB Server Version``   software version of the ``fdbserver``                      command line argument to snap create binary
+``CreationTime``                  current system date and time                               time obtained by calling the system time
+``FoundationDB Cluster File``     cluster file which has cluster-name, magic and             read from the location of the cluster file location
+                                  the list of coordinators.                                  mentioned in the command line arguments. Command
+                                                                                             line arguments of fdbserver can be accessed from
+                                                                                             /proc/$PPID/cmdline
+``Config Knobs``                  command line arguments passed to ``fdbserver``             available from command line arguments of ``fdbserver``
+                                                                                             or from foundationdb.conf
+``IP Address + Port``             host address and port information of the ``fdbserver``     available from command line arguments of ``fdbserver``
+                                  that is invoking the snapshot
+``LocalityData``                  machine id, zone id or any other locality information      available from command line arguments of ``fdbserver``
+``Name for the snapshot file``    Recommended name for the disk snapshot                     cluster-name:ip-addr:port:UID
+================================  ========================================================   ========================================================
+
+snapshot create binary will not be invoked on processes which does not have any persistent data (for eg: Cluster Controller or Master or MasterProxy). Since these processes are completely stateless there is no need for any state information from them. But, if there are specialized configuration knobs used for one of these stateless processes then they need to be backedup and restored externally.
+
+Restore Steps
+==============
+Restore is the process of building up the cluster from the snapshotted disk images. Here is list of steps for the restore process:
```

Comment:
I'm not sure why but this sentence looks bold and italic in Github rendering[1].

[1] https://github.com/apple/foundationdb/blob/abcd76b403cf2999c2f5a24312c3b7170e1ddf9a/documentation/sphinx/source/disk-snapshot-backup.rst

Reply:
It is because of a no line gap between the header and the paragraph, fixed it.